main.py

```diff
@@ -85,34 +125,58 @@ def combo_answer(choice):
     global selected_app 
     selected_app = choice
 
-def make_borderless(check = None):
+def combo_answer_resolution(choice):
+    global selected_resolution 
+    selected_resolution = choice
+
+def combo_answer_display(choice):
+    global selected_monitor
+    selected_monitor = choice
+    label.configure(text="Display Resolution is " + str(monitors[choice].width) + 'x' + str(monitors[choice].height))
+
+def make_borderless(app_name = None, resolution_string = None, monitor_name = None):
     global selected_app, windowList, saveList, temp_win_height, temp_win_width
     
+    app_name = app_name or selected_app
+    resolution_string = resolution_string or selected_resolution
+    monitor_name = monitor_name or selected_monitor
+    
     hwnd = None
     for win_hwnd, win_title in windowList:
-        if win_title.startswith(selected_app):
+        if win_title.startswith(app_name):
             hwnd = win_hwnd
             break
     
     if hwnd is None:
         return
 
-    if check == None:
-        left, top, right, bottom = win32gui.GetWindowRect(hwnd)
-        temp_win_height = bottom - top
-        temp_win_width = right - left
+    # I cannot figure out what this is useful for, so I removed it for now. Feel free to reverse this if I missed something.
```

Comment:
Okay, so I used that check to store the height and width of the app before making it borderless. That was used to restore the app as it was when "Undo" was pressed. It's a bit of hack job which stores values for the recent conversion but is much needed for some legacy games where "Undo"-ing didn't restore them with correct dimensions.

Reply:
Since the app is using a dict in save file, I guess I can make it store a prev_dimension or something.

Reply:
Ah, makes sense! That sounds like a decent solution yeah.